integration-tests/quartz/src/main/resources/application.properties

```diff
@@ -15,3 +15,7 @@ quarkus.flyway.migrate-at-start=true
 quarkus.flyway.baseline-on-migrate=true
 quarkus.flyway.baseline-version=1.0
 quarkus.flyway.baseline-description=Quartz
+
+# Register de LoggingJobHistoryPlugin for testing
+quarkus.quartz.plugin.jobHistory.class=org.quartz.plugins.history.LoggingJobHistoryPlugin
+quarkus.quartz.plugin.jobHistory.jobSuccessMessage=Job [{1}.{0}] execution complete and reports: {8}
```

Comment:
Nice. 

Do you think we can be able to add a line or two in the [Quartz guide](https://github.com/quarkusio/quarkus/blob/ed51c0e624d1e89ebaf9ee6f1fd971e8056689b9/docs/src/main/asciidoc/quartz.adoc) about configuring listeners and plugins?

Reply:
See https://github.com/quarkusio/quarkus/issues/10962#issuecomment-664231324